build/KoreBuild.sh

```diff
@@ -152,7 +152,13 @@ export ReferenceAssemblyRoot=$NUGET_PACKAGES/netframeworkreferenceassemblies/$ne
 makeFileProj="$scriptRoot/KoreBuild.proj"
 msbuildArtifactsDir="$repoFolder/artifacts/msbuild"
 msbuildResponseFile="$msbuildArtifactsDir/msbuild.rsp"
-msbuildLogFile="$msbuildArtifactsDir/msbuild.binlog"
+msBuildLogArgument=""
+
+if [ ! -z "$KOREBUILD_ENABLE_BINARY_LOG" ]; then
```

Comment:
@smitpatel could you file a follow-up issue to make this and `KOREBUILD_SKIP_RUNTIME_INSTALL` consistent with the other environment variables used in these scripts.